apps/web/src/components/WalletConnection/ConnectWalletButton.tsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable */
+import React, { useState } from 'react'
+import { useWeb3React } from '@web3-react/core'
+import { InjectedConnector } from '@web3-react/injected-connector'
+import { BoxLayout, Text, StackLayout } from '@bloxifi/ui'
+
+export const ConnectWalletButton = () => {
+  const [error, setError] = useState(false)
+  const { activate, active, account, deactivate } = useWeb3React()
+
+  async function connect() {
+    try {
+      await activate(new InjectedConnector({}), undefined, true)
+      setError(false)
+    } catch (e) {
+      setError(true)
+    }
+  }
+
+  function disconnect() {
+    try {
+      deactivate()
+    } catch (error) {
+      console.log(error)
```

Comment:
we should avoid console.log, maybe try `throw new Error(error)` we do have a lint for this (I think)